CONTRIBUTING.md

```diff
@@ -1,38 +1,58 @@
+# Contributing
+
 Welcome to firejail, and thank you for your interest in contributing!
 
-# Opening an issue:
-We welcome issues, whether to ask a question, provide information, request a new profile or
-feature, or to report a suspected bug or problem.
+## Opening an issue
```

Comment:
OT: We need a "Starting a Dicussion" section.

Reply:
@rusty-snake [on Mar 26](https://github.com/netblue30/firejail/pull/5757#discussion_r1148572202):

> OT: We need a "Starting a Dicussion" section.

Yes; I have a WIP branch which adds a list related to that to CONTRIBUTING.md.

But this PR is mostly about formatting.